appinventor/buildserver/src/com/google/appinventor/buildserver/DexExecTask.java

```diff
@@ -100,42 +103,40 @@ private boolean preDexLibraries(List<File> inputs) {
             return true;
         }
 
-        synchronized(semaphore) {
+        synchronized (semaphore) {
 
             final int count = inputs.size();
-            boolean allSuccessful = true;
-            for (int i = 0 ; i < count; i++) {
+            for (int i = 0; i < count; i++) {
                 File input = inputs.get(i);
                 if (input.isFile()) {
                     // check if this libs needs to be pre-dexed
                     String fileName = getDexFileName(input);
                     File dexedLib = new File(mDexedLibs, fileName);
                     String dexedLibPath = dexedLib.getAbsolutePath();
 
-                    if (dexedLib.isFile() == false/*||
+                    if (!dexedLib.isFile()/*||
                                                     dexedLib.lastModified() < input.lastModified()*/) {
 
                         System.out.println(
-                            String.format("Pre-Dexing %1$s -> %2$s",
-                              input.getAbsolutePath(), fileName));
+                                String.format("Pre-Dexing %1$s -> %2$s",
```

Comment:
Just one minor code style issue here and in 131-132: The continuing indentation is 4 spaces, not 8.